### PR TITLE
222 add GitHub action integration guides to kobiton docs

### PR DESCRIPTION
**Pull request by Hikkigaya**
### Summary
Two pages added for for GitHub Action with an additional index page.

### Related PRs, issues, or features (optional)

- 222

### Metadata

- [x] Adds new file(s)
- [ ] Edits existing file(s)
- [ ] Removes file(s)

### PR contributor checklist

- [x] My PR follows the [Kobiton Docs contributor guidelines](https://github.com/kobiton/docs/blob/main/CONTRIBUTE.adoc), meaning:

    - My content contains correct spelling and grammar.
    - My directories and files are [named](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-and-file-names) and [structured](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#directory-structure) correctly.
    - My style and voice follows the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/brand-voice-above-all-simple-human).
    - I added all new pages to their section's [`nav.adoc` file](https://github.com/kobiton/docs/blob/main/CONTRIBUTING.md#configure-navigation-in-navadoc).
    - I removed all localizations (like `en-us`) from my URLs.

### CODE REVIEW

File: docs/modules/integrations/pages/github-action/upload-an-app.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+You'll need the following:
+
+* A Kobiton account. To create a Kobiton account visit link:https://portal.kobiton.com/register[Kobiton].
+* Kobiton Username and API key. To get your Kobiton Username and API Key, follow instructions at `IV. Configure Test Script` section on link:https://kobiton.com/blog/parallel-testing-with-selenium-webdriver/[Kobiton blog].
+
+== Upload the app
+
+To upload an app to Kobiton using GitHub Actions, follow the instruction on our link:https://github.com/marketplace/actions/kobiton-upload-mobile-app#kobiton-action-for-upload-mobile-app[official GutHub Marketplace Actions] page.
+
```
Comment (tungmhoang) [inline docs/modules/integrations/pages/github-action/upload-an-app.adoc:18]: Typo: `GitHub` not `GutHub`

File: docs/modules/integrations/pages/github-action/run-appium-script.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+You'll need the following:
+
+* A Kobiton account. To create a Kobiton account visit link:https://portal.kobiton.com/register[Kobiton].
+* Kobiton Username and API key. To get your Kobiton Username and API Key, follow instructions at `IV. Configure Test Script` section on link:https://kobiton.com/blog/parallel-testing-with-selenium-webdriver/[Kobiton blog].
+* `EXECUTOR_USERNAME` and `EXECUTOR_PASSWORD`. To get your executor username and password, contact Kobiton support.
+
+== Run the Appium script
+
+To run Appium script on Kobiton devices using GitHub Actions, follow the instruction on our link:https://github.com/marketplace/actions/kobiton-execute-appium-test[official GutHub Marketplace Action for Appium script] page.
```
Comment (tungmhoang) [inline docs/modules/integrations/pages/github-action/run-appium-script.adoc:18]: Typo: `GitHub` not `GutHub`

File: docs/modules/integrations/pages/github-action/run-appium-script.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Learn how to utilize Kobiton automation testing with Appium script and GitHub Action.
+
+NOTE: For more information, see link:https://github.com/marketplace/actions/kobiton-execute-appium-test[Official GitHub Marketplace Kobiton Action for Appium].
+
+== Before you start
+
+You'll need the following:
+
+* A Kobiton account. To create a Kobiton account visit link:https://portal.kobiton.com/register[Kobiton].
```
Comment (tungmhoang) [inline docs/modules/integrations/pages/github-action/run-appium-script.adoc:12]: Remove this line as it is kind of the same with the second bullet point.

File: docs/modules/integrations/pages/github-action/upload-an-app.adoc [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+Learn how to upload an app to your app repository using GitHub Action.
+
+NOTE: For more information, see link:https://github.com/marketplace/actions/kobiton-upload-mobile-app#setup-workflow-and-action[Official GitHub Marketplace Kobiton Action].
+
+== Before you start
+
+You'll need the following:
+
+* A Kobiton account. To create a Kobiton account visit link:https://portal.kobiton.com/register[Kobiton].
```
Comment (tungmhoang) [inline docs/modules/integrations/pages/github-action/upload-an-app.adoc:12]: Remove this line as it is kind of the same with the second bullet point.

File: docs/modules/integrations/pages/github-action/upload-an-app.adoc [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+* Kobiton Username and API key. To get your Kobiton Username and API Key, follow instructions at `IV. Configure Test Script` section on link:https://kobiton.com/blog/parallel-testing-with-selenium-webdriver/[Kobiton blog].
```
Comment (tungmhoang) [inline docs/modules/integrations/pages/github-action/upload-an-app.adoc:13]: Try to link to pages within the same site. In this case it should be linked to `profile/manage-your-api-credentials#_get_an_api_key`

File: docs/modules/integrations/pages/github-action/run-appium-script.adoc [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+* Kobiton Username and API key. To get your Kobiton Username and API Key, follow instructions at `IV. Configure Test Script` section on link:https://kobiton.com/blog/parallel-testing-with-selenium-webdriver/[Kobiton blog].
```
Comment (tungmhoang) [inline docs/modules/integrations/pages/github-action/run-appium-script.adoc:13]: Try to link to pages within the same site. In this case it should be linked to `profile/manage-your-api-credentials#_get_an_api_key`

**Review by tungmhoang — CHANGES_REQUESTED** [review]
@Hikkigaya some comments that need your work.

File: docs/modules/integrations/pages/github-action/upload-an-app.adoc (hunk 1 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-action/upload-an-app.adoc:18]: Corrected.

File: docs/modules/integrations/pages/github-action/run-appium-script.adoc (hunk 2 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-action/run-appium-script.adoc:18]: Corrected.

File: docs/modules/integrations/pages/github-action/run-appium-script.adoc (hunk 3 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-action/run-appium-script.adoc:12]: 1st bullet point " A Kobiton account. To create a Kobiton account visit link:https://portal.kobiton.com/register[Kobiton]." removed.

File: docs/modules/integrations/pages/github-action/upload-an-app.adoc (hunk 4 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-action/upload-an-app.adoc:12]: 1st bullet point " A Kobiton account. To create a Kobiton account visit link:[https://portal.kobiton.com/register[Kobiton]](https://portal.kobiton.com/register%5BKobiton%5D)." removed.

File: docs/modules/integrations/pages/github-action/upload-an-app.adoc (hunk 5 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-action/upload-an-app.adoc:13]: Linked within the same site.

File: docs/modules/integrations/pages/github-action/run-appium-script.adoc (hunk 6 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-action/run-appium-script.adoc:13]: Linked within the same site

**Comment by tungmhoang** [comment]
@erincbailey For this PR, I would like your opinion on 2 options:
- Either have pages with minimal content, and in these pages include links to the Github marketplace with the integration instructions.
- Or link directly to the Github marketplace from our navigation bar.
I included both options here in the preview. We can then discuss which one to keep: http://10.2.7.208/integrations/github-action/

File: docs/modules/integrations/pages/github-actions/upload-an-app.adoc [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+
+== Before you start
+
+You'll need the following:
+
+* Kobiton username and API key. To obtain them, follow the instructions in xref:profile:manage-your-api-credentials.adoc#_get_an_api_key[this guide].
+
+== Upload the app
+
+To upload an app to Kobiton using GitHub Actions, follow the instruction on our link:https://github.com/marketplace/actions/kobiton-upload-mobile-app#kobiton-action-for-upload-mobile-app[official GitHub Marketplace Actions] page.
```
Comment (erincbailey) [inline docs/modules/integrations/pages/github-actions/upload-an-app.adoc:14]: Suggestion to reduce words and focus on what it's called there: 

Follow the instructions posted on GitHub Marketplace for the link:https://github.com/marketplace/actions/kobiton-upload-mobile-app#kobiton-action-for-upload-mobile-app[Kobiton Action for Upload Mobile App].

Comment (erincbailey) [reply docs/modules/integrations/pages/github-actions/upload-an-app.adoc:14]: Also - what is our policy or what should our policy be for opening in a new window - should this open in a new window? (And what happens if we do that on the widget?)

File: docs/modules/integrations/pages/github-actions/run-appium-script.adoc [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+== Before you start
+
+You'll need the following:
+
+* Kobiton username and API key. To obtain them, follow the instructions in xref:profile:manage-your-api-credentials.adoc#_get_an_api_key[this guide].
+* `EXECUTOR_USERNAME` and `EXECUTOR_PASSWORD`. The executor username and password are the same as the Kobiton username and API key.
+
+== Run the Appium script
+
+To run Appium script on Kobiton devices using GitHub Actions, follow the instruction on our link:https://github.com/marketplace/actions/kobiton-execute-appium-test[official GitHub Marketplace Actions for Appium script] page.
```
Comment (erincbailey) [inline docs/modules/integrations/pages/github-actions/run-appium-script.adoc:15]: Suggested: 
Follow the instructions posted on GitHub Marketplace for the link:https://github.com/marketplace/actions/kobiton-execute-appium-test[Kobiton Action for Execute Appium Test].

File: docs/modules/integrations/pages/github-actions/run-appium-script.adoc [hunk 9]
```diff
@@ -0,0 +1,15 @@
+= Run Appium script using GitHub Actions
+:navtitle: Run Appium script
+
+Learn how to utilize Kobiton automation testing with Appium script and GitHub Actions.
```
Comment (erincbailey) [inline docs/modules/integrations/pages/github-actions/run-appium-script.adoc:4]: Suggestion: 
Learn how to run an Appium script on Kobiton with GitHub Actions.

**Review by erincbailey — CHANGES_REQUESTED** [review]

File: docs/modules/integrations/pages/github-actions/upload-an-app.adoc (hunk 7 above)
Comment (tungmhoang) [reply docs/modules/integrations/pages/github-actions/upload-an-app.adoc:14]: I'll update the policy somewhere else, but let me try opening this on a new window and see what happens on the widget.

File: docs/modules/integrations/pages/github-actions/run-appium-script.adoc (hunk 8 above)
Comment (tungmhoang) [reply docs/modules/integrations/pages/github-actions/run-appium-script.adoc:15]: @Hikkigaya can you work on this?

File: docs/modules/integrations/pages/github-actions/run-appium-script.adoc (hunk 9 above)
Comment (tungmhoang) [reply docs/modules/integrations/pages/github-actions/run-appium-script.adoc:4]: @Hikkigaya can you work on this?

File: docs/modules/integrations/pages/github-actions/upload-an-app.adoc (hunk 7 above)
Comment (tungmhoang) [reply docs/modules/integrations/pages/github-actions/upload-an-app.adoc:14]: @Hikkigaya please work on the wording change and open the link in the new window.

Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-actions/upload-an-app.adoc:14]: Okay, will do

File: docs/modules/integrations/pages/github-actions/run-appium-script.adoc (hunk 8 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-actions/run-appium-script.adoc:15]: Sure

File: docs/modules/integrations/pages/github-actions/run-appium-script.adoc (hunk 9 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-actions/run-appium-script.adoc:4]: Sure

File: docs/modules/integrations/pages/github-actions/upload-an-app.adoc (hunk 7 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-actions/upload-an-app.adoc:14]: Updated

File: docs/modules/integrations/pages/github-actions/run-appium-script.adoc (hunk 8 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-actions/run-appium-script.adoc:15]: Updated

File: docs/modules/integrations/pages/github-actions/run-appium-script.adoc (hunk 9 above)
Comment (Hikkigaya) [reply docs/modules/integrations/pages/github-actions/run-appium-script.adoc:4]: Updated

**Review by tungmhoang — APPROVED** [review]

**Comment by tungmhoang** [comment]
Changes ready for your final review @erincbailey : http://10.2.7.208/integrations/github-actions/

**Review by tungmhoang — APPROVED** [review]

**Review by erincbailey — APPROVED** [review]
approved

**Pull request merged**